Owncloud iOs Client/Tabs/FileTab/Share/ShareMainViewController.m

```diff
@@ -505,19 +505,24 @@ - (UITableViewCell *) getCellOfFileOrFolderInformationByTableView:(UITableView *
     shareFileCell.fileSize.hidden = self.sharedItem.isDirectory;
     shareFileCell.folderName.hidden = !self.sharedItem.isDirectory;
     
-    //Add long press event
-    UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(didLongPressPrivateLinkButton:)];
-   // longPress.minimumPressDuration = 3; //seconds
-    longPress.delegate = self;
-    [shareFileCell.privateLinkButton addGestureRecognizer:longPress];
-    
-    //Add tap event
-    UITapGestureRecognizer *tapGesture = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(didTapPrivateLinkButton)];
-    tapGesture.numberOfTapsRequired = 1;
-    tapGesture.numberOfTouchesRequired = 1;
-    [shareFileCell.privateLinkButton addGestureRecognizer:tapGesture];
-    
-    shareFileCell.privateLinkButton.tag = -1;
+    if (self.sharedItem.ocPrivatelink && !([self.sharedItem.ocPrivatelink isEqual:@""])) {
+        //Add long press event
+        UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(didLongPressPrivateLinkButton:)];
+        // longPress.minimumPressDuration = 3; //seconds
```

Comment:
It seems that this comment wasn't used before neither now.
From the apple documentation:

> **Discussion**
> The time interval is in seconds. The default duration is is 0.5 seconds.

So if this isn't used you can remove it.